OneCommunityGlobal/HGNRest
# Sriram: added new `inventoryModifiedDatetime` field and related updates

# Description
![image](https://github.com/user-attachments/assets/fe29123e-d9ec-4e17-9e92-89ce832488b2)

## Related PRS (if any):
This  PR is related to [this](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2874)  frontend PR.
…

## Main changes explained:
- Added `inventoryModifiedDatetime` field
- Added support to update the field when inventory changes
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Follow instructions on frontend PR.
